symphony-bdk-core/pom.xml

```diff
@@ -205,7 +210,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>${codegen.spec.base}/pod/pod-api-public.yaml</inputSpec>
+                            <inputSpec>${codegen.spec.base}/pod/pod-api-public-deprecated.yaml</inputSpec>
```

Comment:
This is because the message suppression service is only declared in the public-deprecated yaml even if it is not marked as deprecated in the official documentation : https://developers.symphony.com/restapi/reference#suppress-message